src/Serilog.Formatting.Compact/Serilog.Formatting.Compact.csproj

```diff
@@ -1,30 +1,47 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
-  <PropertyGroup>
-    <Description>A simple, compact JSON-based event format for Serilog.</Description>
-    <VersionPrefix>1.1.0</VersionPrefix>
-    <Authors>Serilog Contributors</Authors>
-    <TargetFrameworks>net452;netstandard1.1;netstandard2.0</TargetFrameworks>
-    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
-    <GenerateDocumentationFile>true</GenerateDocumentationFile>
-    <AssemblyName>Serilog.Formatting.Compact</AssemblyName>
-    <AssemblyOriginatorKeyFile>../../assets/Serilog.snk</AssemblyOriginatorKeyFile>
-    <SignAssembly>true</SignAssembly>
-    <PublicSign Condition=" '$(OS)' != 'Windows_NT' ">true</PublicSign>
-    <PackageId>Serilog.Formatting.Compact</PackageId>
-    <PackageTags>serilog;json</PackageTags>
-    <PackageIconUrl>http://serilog.net/images/serilog-extension-nuget.png</PackageIconUrl>
-    <PackageProjectUrl>https://github.com/serilog/serilog-formatting-compact</PackageProjectUrl>
-    <PackageLicenseExpression>Apache-2.0</PackageLicenseExpression>
-  </PropertyGroup>
+    <PropertyGroup>
+        <Description>A simple, compact JSON-based event format for Serilog.</Description>
+        <VersionPrefix>3.0.0</VersionPrefix>
+        <Authors>Serilog Contributors</Authors>
+        <TargetFrameworks>net6.0;netstandard2.0</TargetFrameworks>
```

Comment:
It might be best for this to track the Serilog 3 TFMs, to avoid unnecessary dependencies on .NET 4.6.2, etc.:

```xml
    <TargetFrameworks Condition=" '$(OS)' == 'Windows_NT'">net462;net471</TargetFrameworks>
    <TargetFrameworks>$(TargetFrameworks);netstandard2.1;netstandard2.0;net5.0;net6.0;net7.0</TargetFrameworks>
```